### ISSUE TITLE
[Settings]: At 400% Scaling, Text and Controls are not properly visible. 

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#11261]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Prerequisites:**

1. Open Settings app using 'Windows + i' Key.
2. Navigate to System and activate it.
3. Then Set Display Resolution to 1280 x 1024.
4. Then Navigate to System>Display> Scale
5. Inside Scale page, Set Custom Scaling at 400 and Sign out 

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Verify that whether user can use app properly.

**User Experience:**
Low vision user will be impacted here as content is not properly visible as result user will find difficulty in using the app.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/reflow

**Attachments:**
[At 400% Scaling, Text and Controls are not properly visible..zip](https://github.com/microsoft/terminal/files/7185544/At.400.Scaling.Text.and.Controls.are.not.properly.visible.zip)

### Expected Behavior

At 400% Scaling, Text and Controls should be properly visible.

### Actual Behavior

At 400% Scaling, Text and Controls are not properly visible. User have move up and down to fully see and read Control and Text name.

**Comment by v-rpundir** [comment]
Closing the bug as it is fixed as per update in Desktop Reflow MAS Guidelines t.ly/vkzK

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
